### PR TITLE
implement the markautonomous subcommand

### PR DESCRIPTION
**Pull request by charleszheng44**
1. implement the `markautonomous` subcommand
2. fix golint warning
3. revise the tutorial based on `markautonomous`
4. extract the common function

### CODE REVIEW

File: docs/tutorial/yurtctl.md [hunk 1]
```diff
@@ -119,18 +124,23 @@ us-west-1.192.168.0.88   Ready    <none>   19h   v1.14.8-aliyun.1
...
 Labels:             alibabacloud.com/is-edge-worker=false
 ```
 
+4. Same as before, we make desired edge nodes autonomous:
+```bash
+$ _output/bin/yurtctl markautonomous
+I0602 11:22:05.610222   89160 markautonomous.go:149] mark us-west-1.192.168.0.88 as autonomous
+```
+
 4. When the OpenYurt cluster contains cloud nodes, yurt controller manager will be deployed on the cloud node (in this case, the node `us-west-1.192.168.0.87`):
```
Comment (Fei-Guo) [inline docs/tutorial/yurtctl.md:144]: 4. -> 5.

File: pkg/yurtctl/cmd/markautonomous/markautonomous.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+			if err := co.Validate(); err != nil {
+				klog.Fatalf("markautonomous option is invalid: %s", err)
+			}
+			if err := co.RunMarkAutonomous(); err != nil {
+				klog.Fatalf("fail to make nodes autonomous: %s", err)
+			}
+		},
+	}
+
+	cmd.Flags().Bool("mark-all-edge-nodes", true,
```
Comment (Fei-Guo) [inline pkg/yurtctl/cmd/markautonomous/markautonomous.go:65]: Can we remove this flag? If autonomous-nodes string is empty, we assume it means all edge nodes.

File: docs/tutorial/yurtctl.md (hunk 1 above)
Comment (charleszheng44) [reply docs/tutorial/yurtctl.md]: changed

File: pkg/yurtctl/cmd/markautonomous/markautonomous.go (hunk 2 above)
Comment (charleszheng44) [reply pkg/yurtctl/cmd/markautonomous/markautonomous.go]: removed

**Pull request merged**